packages/augur-ui/src/modules/global-chat/components/global-chat.tsx

```diff
@@ -28,7 +30,7 @@ export const GlobalChat = ({ provider, whichChatPlugin }: GlobalChatProps) => {
           mute
           popupChat
           showEmoji
-          colorTheme="#0E0E0F"
+          colorTheme={theme !== THEMES.TRADING ? '#D1D1D9' : '#0E0E0F'}
```

Comment:
Is there any better way to do this with just CSS? we should avoid hard coding styling hex values into the html

Reply:
Will submit a PR to 3box and will fix it as soon as it's published.